### ISSUE TITLE
[dev.icinga.com #12560] Don't send Flapping* notifications when downtime is active

### ISSUE THREAD

**Issue by icinga-migration** [Icinga/icinga2#4574]
This issue has been migrated from Redmine: https://dev.icinga.com/issues/12560

**Created by elasto on 2016-08-26 17:00:29 +00:00**

Assignee: _mfriedrich_
Status: _Resolved_  (closed on _2016-11-10 13:16:36 +00:00_)
Target Version: _2.6.0_
Last Update: _2016-12-12 12:37:12 +00:00 (in Redmine)_

    Icinga Version: 2.5.3
    Backport?: Not yet backported
    Include in Changelog: 1

---

Hello.

CentOS 6.8 x86\_64

\[1472199752\] EXTERNAL COMMAND: SCHEDULE\_HOST\_SVC\_DOWNTIME;example.com;1472199744;1472203344;1;0;0;icingaadmin;ack
\[1472202218\] HOST DOWNTIME ALERT: example.com;STARTED; Checkable has entered a period of scheduled downtime.

\[1472202218\] HOST ALERT: example.com;DOWN;SOFT;1;PING CRITICAL - Packet loss = 100%
\[1472202276\] HOST ALERT: example.com;DOWN;SOFT;2;PING CRITICAL - Packet loss = 100%
\[1472202335\] HOST ALERT: example.com;DOWN;HARD;1;PING CRITICAL - Packet loss = 100%

\[1472202593\] HOST NOTIFICATION: icingaadmin;example.com;FLAPPINGSTART (UP);mail-host-notification;PING WARNING - Packet loss = 81%, RTA = 0.36 ms;
\[1472202776\] HOST NOTIFICATION: icingaadmin;example.com;FLAPPINGEND (UP);mail-host-notification;PING OK - Packet loss = 0%, RTA = 0.36 ms;

\[1472203401\] HOST DOWNTIME ALERT: example.com;STOPPED; Checkable has exited from a period of scheduled downtime.

1. HOST DOWNTIME started after 2466 seconds after EXTERNAL COMMAND: SCHEDULE\_HOST\_SVC\_DOWNTIME
2. Notifications was sent when downtime is active

**Changesets**

_2016-11-10 13:16:02 +00:00_ by _mfriedrich_ 7e0c48643b3f6985ed584648dfab1ec485678bc2

    Fix Flapping{Start,End} notifications in SOFT states or downtimes
    
    fixes #12560
    fixes #12892

_2016-11-17 12:29:27 +00:00_ by _mfriedrich_ 0610652e369e1093dfe80150bd9c13ee894fe351

    Fix Flapping{Start,End} notifications in SOFT states or downtimes
    
    fixes #12560
    fixes #12892

---

**Relations**:

* relates [#12892](https://dev.icinga.com/issues/12892)
